cfme/tests/test_rest.py

```diff
@@ -1,9 +1,12 @@
 # -*- coding: utf-8 -*-
 """This module contains REST API specific tests."""
 import pytest
+import fauxfactory
+import utils.error as error
 
 from cfme import test_requirements
 from cfme.rest import vm as _vm
+from cfme.rest import arbitration_settings as _arbitration_settings
```

Comment:
Why import it with an underscore? Just to keep with the previous one? I don't see a need, but feel free to correct me.